cli/standalone.rs

```diff
@@ -74,9 +76,10 @@ pub async fn extract_standalone(
 ) -> Result<Option<(Metadata, eszip::EszipV2)>, AnyError> {
   let current_exe_path = current_exe()?;
 
-  let file = tokio::fs::File::open(current_exe_path).await?;
+  let file = std::fs::File::open(current_exe_path)?;
 
-  let mut bufreader = tokio::io::BufReader::new(file);
+  let mut bufreader =
+    deno_core::futures::io::BufReader::new(AllowStdIo::new(file));
```

Comment:
Why this change?

Reply:
See https://github.com/denoland/eszip/pull/103

Reply:
Makes sense, thanks!